### ISSUE TITLE
Drag and drop an event on the same day

### ISSUE THREAD

**Issue by cosf87** [mattlewis92/angular-calendar#1237]
**Describe the bug**
On calendar month view, If you drag an event and drop it on the same day, day click event is fired after drop event.
It works fine when you drag and drop an event from one day to another day, because only drop event is fired.

**Minimal reproduction of the problem with instructions**
On 'Kitchen sink' demo, drag and drop an event on the same day

**Screenshots**

**Versions**

- `@angular/core`: 9.1.2
- `angular-calendar`: 0.28.11
- Browser name and version: Google Chrome 80.0.3987.163

**Comment by mattlewis92** [comment]
Thanks for reporting, I will get that fixed up in the next version. As a workaround you can add this to your global apps CSS:
```
.cal-month-view .cal-cell .cal-event {
  pointer-events: all !important;
}
```

**Comment by mattlewis92** [comment]
I just released `0.28.13` with a fix, thanks for reporting! 😄
